Added flags for postgres forwarding and Docker commands (#12)
PolideaInternal/airflow-breeze: README.md

./run_environment.sh
```

## Forwarding webserver port to Airflow's UI

If you want to forward a port for using the webserver, use the --forward-webserver-port flag:

```
./run_environment.sh --forward-webserver-port 8080
```

Note that you need to start the webserver manually with `airflow webserver` command.

> ## Forwarding postgres port to Airflow's Postgres DB

Reviewer: ```suggestion
## Forwarding Postgres port to Airflow's Postgres DB
```

> If you want to forward a port for using the webserver, use the --forward-postgres-port flag:

Reviewer: ```suggestion
If you want to forward a port for remotely accessing the database, use the --forward-postgres-port flag:
```

-P, --python <PYTHON_VERSION>
        Python virtualenv used by default. One of ('2.7', '3.5', '3.6'). [2.7]

-f, --forward-webserver-port <PORT_NUMBER>
        Optional - forward the port PORT_NUMBER to airflow's webserver (you must start
        the server with 'airflow webserver' command manually).

-F, --forward-postgres-port <PORT_NUMBER>
        Optional - forward the port PORT_NUMBER to airflow's Postgres database. You can

>         login to the database as root user with airflow password. Database of airflow

Reviewer: ```suggestion
        login to the database as the user "root" with password "airflow". Database of airflow
```

>         is named airflow/airflow.db

Reviewer: ```suggestion
        is named "airflow/airflow.db".
```
